java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -1660,7 +1660,7 @@ public void fcall_readonly_function() {
 
         assertEquals(libName, clusterClient.functionLoad(code, false).get());
         // let replica sync with the primary node
-        assertEquals(1L, clusterClient.wait(1L, 3000L).get());
+        assertEquals(1L, clusterClient.wait(1L, 10000L).get());
```

Comment:
Please don't increase that.
If replica can't sync with master in 3 sec - we have a trouble

Reply:
I will revert. 
That being said - that's not our responsibility to ensure realtime feedback for Valkey 8.